pkg/reexec/reexec_test.go

```diff
@@ -0,0 +1,54 @@
+package reexec
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func init() {
+	Register("reexec", func() {
+		panic("Return Error")
+	})
+	Init()
+}
+
+func TestRegister(t *testing.T) {
+	defer func() {
+		if r := recover(); r != nil {
+			require.Equal(t, `reexec func already registered under name "reexec"`, r)
+		}
+	}()
+	Register("reexec", func() {})
+}
+
+func TestCommand(t *testing.T) {
+	cmd := Command("reexec")
+	w, err := cmd.StdinPipe()
+	require.NoError(t, err, fmt.Sprintf("Error on pipe creation: %v", err))
```

Comment:
It's not necessary to use `fmt.Sprintf` for the last arguments to `require.NoError`. It accepts printf formatting directly. You could do:

```go
require.NoError(t, err, "Error on pipe creation: %v", err)
```